apps/frontend/src/libs/components/table/styles.module.css

```diff
@@ -37,8 +41,10 @@
 
 .table-data {
 	min-height: 60px;
+	overflow: hidden;
 	font-weight: 400;
 	line-height: 150%;
+	text-overflow: ellipsis;
 	word-wrap: normal;
```

Comment:
Now sure if we want this, we won't be able to view the text in other ways

Reply:
Ok, so what should I do with it then? If we're leaving it as it is, then text will overlap.